src/content/developers/docs/gas/index.md

```diff
@@ -91,13 +99,15 @@ If you want to monitor gas prices so you are able to send your ETH for less you
 - [Ethereum Gas Explained](https://defiprime.com/gas)
 - [Is Ethereum more expensive to use as price rises?](https://docs.ethhub.io/questions-about-ethereum/is-ethereum-more-expensive-to-use-as-price-rises/)
 - [Reducing the gas consumption of your Smart Contracts](https://medium.com/coinmonks/8-ways-of-reducing-the-gas-consumption-of-your-smart-contracts-9a506b339c0a)
+- [Proof of Stake versus Proof of Work](https://blockgeeks.com/guides/proof-of-work-vs-proof-of-stake/)
 
 ## Related Tools {#related-tools}
 
 - [ETH Gas Station](https://ethgasstation.info/) _Consumer oriented metrics for the Ethereum gas market_
 - [Etherscan Gas Tracker](https://etherscan.io/gastracker) _Transaction gas price estimator_
 - [Bloxy Gas Analytics](https://stat.bloxy.info/superset/dashboard/gas/?standalone=true) _Ethereum network gas stats_
 - [Blocknative's Gas Platform](https://www.blocknative.com/gas) _Gas estimation API powered by Blocknative's global mempool data platform_
+- [Gas Now](https://www.gasnow.org) _A tool for forecasting the price of gas on the Ethereum network_
```

Comment:
Reviews about this website have been less than encouraging.
https://chrome.google.com/webstore/detail/gasnow-eth-gasprice-forec/ccnipbjnnkophfgnkmhdongajmkhijbl?hl=en-US
Their browser extension is heavily advertised, and users report that they have to give the extension access to the contents of their clipboard which is an incredible red-flag in regards to security and risk of users having funds stolen.
Would strongly vote against linking to any site with this reputation.